activation/atxdb.go

```diff
@@ -25,6 +26,14 @@ func getNodeAtxPrefix(nodeID types.NodeID) []byte {
 	return []byte(fmt.Sprintf("n_%v_", nodeID.Key))
 }
 
+func getAtxEpochKey(epoch types.EpochID, nodeID types.NodeID) []byte {
```

Comment:
name might indicate you retrieve atxs of an epoch but not for a specific node. though I understand that the signature also speaks..